Ginger/Ginger/SolutionWindows/TreeViewItems/BusinessFlowTreeItem.cs

```diff
@@ -104,6 +107,7 @@ void ITreeViewItem.SetTools(ITreeView TV)
                 }
                 
                 AddItemNodeBasicManipulationsOptions(mContextMenu);
+                TreeViewUtils.AddMenuItem(mContextMenu, "Actions Conversion", ActionsConversionHandler, null, "@Connection_32x32.png");                
```

Comment:
we stopped using static images like- "@Connection_32x32.png", the funcation supports eImageType please use it

Reply:
removed static image